prov/efa/src/rxr/rxr_read.c

```diff
@@ -179,6 +179,46 @@ ssize_t rxr_read_mr_reg(struct rxr_ep *ep, struct rxr_read_entry *read_entry)
 	return 0;
 }
 
+/**
+ * @brief convert descriptor from application for internal use
+ *
+ * The descriptor provided by applicaiton is of type struct efa_mr *,
+ * This function convert it to descriptors for lower provider according
+ * to lower provider type
+ *
+ * @param efa_desc[out] descriptors for lower efa provider.
+ * @param shm_desc[out] descriptors for lower shm provider.
+ * @param desc[in] descriptors provided by application
+ * @param numdesc[in] number of descriptors in the array
```

Comment:
Is @param low_ep_type missing here?
Also I think @desc_out should be here, and you can put the description of efa_desc and shm_desc under it.

Reply:
thank you! fixed in new revision